src/Codeception/Lib/Connector/Yii2.php

```diff
@@ -275,6 +275,16 @@ protected function persistDb()
 
     private function mockAssetManager()
     {
-        $this->app->set('assetManager', Stub::make('yii\web\AssetManager', ['bundles' => false]));
```

Comment:
`@webroot` created via **protected** methods `yii\web\AssetManager::publishFile` and `yii\web\AssetManager::publishDirectory`, but Application uses **public** `yii\web\AssetManager::publish` that used `publishDirectory` and `publishFile` functions.

Why have you stopped to create this directory? Why didn't use `@webroot/assets/` that uses in the real application? You should set correct path to `@webroot/assets/` for that.